nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -279,6 +279,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `services.gnome.experimental-features.realtime-scheduling` option has been removed, as GNOME Shell now [uses rtkit](https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2060). Use `security.rtkit.enable = true;` instead. As before, you will need to have it enabled using GSettings.
 
+- `services.telepathy.enable` has been changed from default `true` to default `false` under `services.x11.desktop-managers.gnome`, one should enable it in their configs if using Empathy or Polaris.
```

Comment:
`services.x11.desktop-managers.gnome` is not a thing. I would recommend talking about GNOME module. Also Polaris → Polari.